ui-tests/__test__/flows/navigate-prompts/navigate-back-to-current-prompt-with-code-attachment.ts

```diff
@@ -0,0 +1,41 @@
+import { Page } from 'playwright/test';
+import { getSelector, waitForAnimationEnd } from '../../helpers';
+import testIds from '../../../../src/helper/test-ids';
+import { closeTab } from '../close-tab';
+import { openNewTab } from '../open-new-tab';
+
+export const navigateBackToCurrentPromptWithCodeAttachment = async (page: Page, skipScreenshots?: boolean): Promise<void> => {
+  await closeTab(page, false, true);
+  await openNewTab(page, false, true);
+
+  await page.locator(`${getSelector(testIds.prompt.input)}`).fill('This is the first user prompt');
+  await page.locator(`${getSelector(testIds.prompt.send)}`).click();
+  await waitForAnimationEnd(page);
+
+  await page.evaluate(() => {
+    const selectedTabId = window.mynahUI.getSelectedTabId();
+    if (selectedTabId != null) {
+      window.mynahUI.addToUserPrompt(
+        selectedTabId,
+        'This is an unsent code attachment',
+        'code',
+      );
+    }
+  });
+  await waitForAnimationEnd(page);
+
+  let promptInput = await page.locator(`${getSelector(testIds.prompt.input)}`);
+  await promptInput.press('ArrowUp');
+  await waitForAnimationEnd(page);
+
+  promptInput = await page.locator(`${getSelector(testIds.prompt.input)}`);
+  await promptInput.press('ArrowDown');
+  await waitForAnimationEnd(page);
+
+  const codeAttachmentContent = (await page.locator(`${getSelector(testIds.prompt.attachment)}`).innerText()).trim();
```

Comment:
add to add `.trim()` because webpack test was failing otherwise as it adds a `\n` at the end, like `'This is an unsent code attachment\n'`

Reply:
Nice workaround. Add a comment above this line with this disclaimer as well for future code clarity.